DSCResources/MSFT_xSQLServerSetup/MSFT_xSQLServerSetup.psm1

```diff
@@ -563,6 +611,12 @@ function Set-TargetResource
         "InstallSharedWOWDir",
         "InstanceDir"
     )
+
+    if($BrowserSvcStartupType -ne $null)
```

Comment:
local variable names should start with a lowercase (as per https://github.com/PowerShell/DscResources/blob/TravisEz13-patch-1/StyleGuidelines.md#variable-names-should-use-camelcase)